File: laravel-demo/app/Http/Controllers/IconsController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class IconsController extends Controller
{
    public function fontAwesome()
    {
        return view('icons.font-awesome', ['title' => 'Font Awesome 5 - Icons']);
    }

    public function ionicons()
    {
        return view('icons.ionicons', ['title' => 'Ionicons - Icons']);
    }

    public function linearicons()
    {
        return view('icons.linearicons', ['title' => 'Linearicons - Icons']);
    }

    public function openiconic()
    {
        return view('icons.openiconic', ['title' => 'Open Iconic - Icons']);
    }

    public function stroke7()
    {
        return view('icons.stroke7', ['title' => 'Stroke Icons 7 - Icons']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/IconsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers
{
    public class IconsController : Controller
    {

         public ActionResult FontAwesome()
        {
            return View("~/Views/Icons/FontAwesome.cshtml");
        }

         public ActionResult Ionicons()
        {
            return View("~/Views/Icons/Ionicons.cshtml");
        }

         public ActionResult Linearicons()
        {
            return View("~/Views/Icons/Linearicons.cshtml");
        }

         public ActionResult Openiconic()
        {
            return View("~/Views/Icons/Openiconic.cshtml");
        }

         public ActionResult Stroke7()
        {
            return View("~/Views/Icons/Stroke7.cshtml");
        }

    }
}
